lib/screens/events/events_repository.dart

```diff
@@ -79,7 +73,9 @@ class APIgetAllEventsRepository implements GetAllEventsRepository {
     if (response.statusCode == 200) {
       Log.i(tag: tag, message: "Request Successful");
       var json = jsonDecode(response.body);
-      return json["data"];
+      List<Events> events;
+      (json["data"] as List).map((e) => events.add((Events.fromJson(e))));
```

Comment:
Careful what you type. And test before pushing the code.
```suggestion
      (json["data"] as List).map((e) => events.add(events.add(Events.fromJson(e))));
```